scripts/cmake_format_all.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+SCRIPTDIR=$(cd $(dirname $0); pwd)
+BASEDIR=$(dirname $SCRIPTDIR)
+
+find $BASEDIR -name CMakeLists.txt  -exec cmake-format -i {} +
+find $BASEDIR/src $BASEDIR/test $BASEDIR/tsl -name '*.cmake' -exec cmake-format -i {} +
```

Comment:
Where is this script executed?

Reply:
In the custom target inside the top-level `CMakeLists.txt`.